Assets/Scripts/Interfaces/IDamageable.cs

```diff
@@ -0,0 +1,11 @@
+using UnityEngine;
+
+public interface IDamageable
+{
+    public float Health { set; get; }
+    public bool Targetable { set; get; }
+    public bool Invincible { set; get; }
```

Comment:
I would make the setter private for these. You don't want to change them from outside

Reply:
can't make them private and still have set, get apparently

Reply:
it's a rule in interfaces apparently